processing/mars_stealer/mars_stealer.py

```diff
@@ -0,0 +1,336 @@
+from fame.core.module import ProcessingModule
+from fame.common.exceptions import ModuleInitializationError
+
+try:
+    import re
+    import json
+    import base64
+    import r2pipe
+    from typing import Any, List
+    from collections import namedtuple
+    from cryptography.hazmat.primitives.ciphers import Cipher, algorithms
+
+    Section = namedtuple("Section",
+                         ["name", "id", "paddr", "size",
+                          "vaddr", "vsize", "perm"])
+    HAVE_CSPARSER = True
+except ImportError as err:
+    print(err)
+    HAVE_CSPARSER = False
+
+
+def unxor(string: List[Any], key: List[Any]) -> str:
+    """Method to unxor obfuscated data from llcppc section"""
+
+    unxored = ""
+
+    for c1, c2 in zip(string, key):
+        unxored += chr(c1 ^ c2)
+
+    return unxored
+
+
+def decrypt_rc4(key: bytes, ciphertext: bytes) -> bytes:
+    """Decrypt RC4 encrypt data"""
+
+    algorithm = algorithms.ARC4(key)
+    cipher = Cipher(algorithm, mode=None)
+    decryptor = cipher.decryptor()
+    cleartext = decryptor.update(ciphertext)
+
+    return cleartext
+
+
+class MarsStealerC2Extractor(ProcessingModule):
+
+    name: str = "mars_stealer_c2"
```

Comment:
```suggestion
    name: str = "mars_stealer"
```

When there is a match, the name of the module is added as a tag so it is a convention to give the module the name of what it is able to recognize.